listener/upload_listener.py

```diff
@@ -218,7 +219,7 @@ def process_upload(upload_data, loop=None):
         rtrn = db_import_system(upload_data["id"], upload_data["account"], upload_data["url"],
                                 vmaas_request, satellite_managed)
         # only give evaluator work if the system's vmaas-call has changed since the last time we did this
-        if rtrn['changed']:
+        if rtrn['changed'] and not DISABLE_OPTIMISATION:
```

Comment:
this doesn't work as expected - if DISABLE_OPTIMISATION is True then message to evaluator should be sent regardless of changed value